Assets/Scripts/Player/Rock.cs

```diff
@@ -0,0 +1,49 @@
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Rock : MonoBehaviour
+{
+    [SerializeField] int dmg = 1;
+    [SerializeField] float range = .8f;
+    [SerializeField] LayerMask enemyLayer;
+    Rigidbody2D rb;
+
+    [SerializeField] UnityEvent destroyedEvent;
+    [SerializeField] UnityEvent createdEvent;
+
+    private void Awake()
+    {
+        rb = GetComponent<Rigidbody2D>();
+        createdEvent.Invoke();
+    }
+
+    public void Lunch(Vector2 direction, float force)
```

Comment:
Fix typo, assuming it was meant to be launch?

Reply:
done